kubedriver/helmclient/client.py

```diff
@@ -175,11 +175,12 @@ def get(self, name, namespace):
                 cmd = self.__helm_cmd('get', "all", name)
         else:
             cmd = self.__helm_cmd('get', name)
-        process_result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        # Keep stdout and stderr separate here so helm warnings to stderr do not cause parsing to fail.
+        process_result = subprocess.run(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         if process_result.returncode == 127:
             raise HelmCommandNotFoundError(f'Helm install command not found: {process_result.stdout}')
```

Comment:
Do we need to add `stderr` to this error message, in case it includes useful information?